paper/paper.md

```diff
@@ -95,7 +95,17 @@ pyOBIS can be used to do super-useful time series analysis for instance, absolut
 * However, necessary precautions to avoid sampling bias must be taken into consideration.
 
 # Conclusion
-
+The pyOBIS python package provides a convenient and efficient way to access and work with taxonomic occurrence records from the Ocean Biodiversity Information System (OBIS).
+With over 107 million occurrence records,
+OBIS is a valuable resource for marine biodiversity data,
+but its accessibility has been a challenge which pyOBIS addresses by offering built-in functions for retrieving data on occurrences, taxons, nodes, checklists, and dataset metadata.
+It enables researchers to download,
+visualize, segment, process, and export data in various formats using its tools or other Python libraries.
+By integrating with sister packages like pyDwcViz,
```

Comment:
Is it worth making this a link to the package?

Reply:
I'm not sure, but I included so that it establishes another link to possible research applications.

Reply:
okay, i'll remove it then.

Reply:
I meant my suggestion to make it a link, sorry for the confusion.

Reply:
oh, my bad. I thought the mention of the package. 🥲